### DISCUSSION TITLE
Awesome work!

### DISCUSSION THREAD

**Discussion by bauerj** [astubenbord/paperless-mobile#46] [General]
Hey,

I've just learned about your app and there are a lot of things I like about it! The scanner is a lot more convenient than in my app.

Keep up the great work and maybe I'll switch to using your version soon 😄

**Comment by JigSawFr** [comment]
@bauerj Don't hesitate to centralize effort with us ! ;)

**Comment by astubenbord** [comment]
Thanks Johann, much appreciated! 😄 

I mean if you're interested I'd be happy about any new collaborators since this is kind of getting out of hand and I currently don't have enough time to keep track of and fix all the issues.

Grüße!

**Reply by bauerj** [reply]
>  currently don't have enough time to keep track of and fix all the issues.

Same here, sadly!
